File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        hash_table = {}

        for num in nums:
            if hash_table.get(num, 0) == 1:
                return True
            else:
                hash_table[num] = 1
        
        return False

File: 0217-contains-duplicate/0217-contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> seen = new HashSet<int>();
        for (int i=0; i<nums.Length; i++) {
            seen.Add(nums[i]);
        }
        return seen.Count != nums.Length;
        
    }
}